SimPEG/EM/Static/DC/ProblemDC.py

```diff
@@ -37,14 +39,55 @@ def fields(self, m=None):
         f[Srcs, self._solutionType] = u
         return f
 
-    def Jvec(self, m, v, f=None):
+    def getJ(self, m, f=None):
+
+        if self.verbose:
+            print("Calculating J and storing")
+
+        self.model = m
 
         if f is None:
             f = self.fields(m)
 
+        self.Jmat = []
+        AT = self.getA()
```

Comment:
it doesn't look like this is used in the below calculation?

Reply:
good point. removed `AT = self.getA()`